### ISSUE TITLE
gerbera looks for taglib-config first

### ISSUE THREAD

**Issue by neheb** [gerbera/gerbera#1700]
Causes an issue at least on debian:

```
-- TagLib version not found: version searched: 1.11, found: /usr/bin/taglib-config: 19: /usr/bin/taglib-config: dpkg-architecture: not found
1.11.1
CMake Error at cmake/FindTaglib.cmake:90 (message):
  Could not find Taglib
```

**Comment by KarlStraussberger** [comment]
That's an installation issue.

`taglib-config` calls `dpkg-architecture` which is not installed on your system.

**Comment by neheb** [comment]
Why should it be? pkgconf works just fine.

**Comment by KarlStraussberger** [comment]
Because `taglib-config` has to be part of a correct taglib-dev installation. If it is not there or not working, this installation is not correct.

**Comment by neheb** [comment]
My point is that taglib comes with a perfectly fine pkgconfig file. pkgconfig should be tried first.

**Comment by KarlStraussberger** [comment]
There are newer versions of `FindTaglib.cmake` around. Feel free to PR an update!

**Comment by whyman** [comment]
Closed in #1855
